pkg/windows/windows.go

```diff
@@ -212,6 +215,7 @@ func New(workerIgnitionEndpoint, clusterDNS, vxlanPort string, instanceInfo *ins
 			log:                    log,
 			defaultShellPowerShell: defaultShellPowershell(conn),
 			platformType:           platformType,
+			runtime:                runtime,
```

Comment:
Is this PR missing code? This isnt being used anywhere, is it?

Reply:
In that case these commits should be within his PR. In most cases, variables and code in general should only be merged in when they are actually being used.